.github/workflows/stable_check.yml

```diff
@@ -53,5 +53,5 @@ jobs:
         run: |
           moon test --target wasm
           moon test --target wasm-gc
-          moon test --target native
-
+          moon test --target js
```

Comment:
@myfreess I disabled the native test becuase this issue https://github.com/moonbitlang/moon/issues/950

And added js test.